### ISSUE TITLE
Deleting Posts Messes Up Post Counter

### ISSUE THREAD

**Issue by oleande-r** [RolePlayGateway/roleplaygateway#76]
As the title says, every time you delete a post the content will in fact disappear, but a roleplay will still say "1 post here" even if nothing is there. If you go to the location that the post was posted in it will say the same, and it is wrong for character profiles as well. The post counter continues to be incorrect even when posting again or refreshing.

**Comment by martindale** [comment]
Nice find.  Can you link an example?

**Comment by oleande-r** [comment]
Yeah, I noticed this happening on my own roleplay. [Here's a link.](https://www.roleplaygateway.com/roleplay/remnants-lost)

At the time of posting this there are only four posts in total, but the post counter does say "5 posts here." due to the fact that I had deleted a post. Going to the location where the post was deleted from ([link here](https://www.roleplaygateway.com/roleplay/remnants-lost/places/beacon-academy)) will say that there is "1 post here" even though there isn't anything there. All of the characters in the roleplay also have an additional post added to their "Seen x times" counter.
